README.md

```diff
@@ -113,90 +113,117 @@ Now we can apply this configuration to the cluster using:
+Name:         demo-certificate-4260521829
+Namespace:    default
+Labels:       <none>
+API Version:  cert-manager.io/v1alpha2
+Kind:         CertificateRequest
+Metadata:
+  Creation Timestamp:  2020-03-11T16:18:19Z
+  Generation:          1
+  Owner References:
+    API Version:           cert-manager.io/v1alpha2
+    Block Owner Deletion:  true
+    Controller:            true
+    Kind:                  Certificate
+    Name:                  demo-certificate
+    UID:                   58c9ef95-338a-41a2-a617-6a397b3cdbb9
+  Resource Version:        14429
+  Self Link:               /apis/cert-manager.io/v1alpha2/namespaces/cert-manager/certificaterequests/demo-certificate-4260521829
+  UID:                     66795d81-5ff7-4e89-9ffc-b7863592b95b
+Spec:
+  Csr:       LS0tLS1CRUdJTiBDRVJUSUZJQ0[...]
+  Duration:  2160h0m0s
+  Issuer Ref:
+    Kind:  Issuer
+    Name:  venafi-tpp-issuer
+Status:
+  Certificate:  LS0tLS1CRUdJTiBDRVJUSU1[...]
+    Last Transition Time:  2020-03-11T16:18:31Z
+    Message:               Certificate fetched from issuer successfully
+    Reason:                Issued
+    Status:                True
+    Type:                  Ready
+Events:
+  Type    Reason             Age   From          Message
+  ----    ------             ----  ----          -------
+  Normal  CertificateIssued  67s   cert-manager  Certificate fetched from issuer successfully
 
-#### Using Firefox
+```
 
-Open your browser and browse to the hostname of your instance.
-You will see an "insecure" warning, this is because for this lab we used a training certificate authority which is not trusted by our computer.
-![Firefox warning](./images/ff-error.png)
-Click on "Advanced" and then to "View Certificate", here you can see the info about the certificate which we just issued.
-![Firefox bypass button](./images/ff-bypass.png)
-If you click "Accept the risk and continue" you will be presented with our "Hello World" service.
-![Firefox certificate details](./images/ff-cert.png)
+Here we see our generated CSR and Certificate (both base64 encoded), as well as detailed info of the status of the CertificateRequest.
 
-#### Using Chrome
-Open your browser and browse to the hostname of your instance.
-You will see an "insecure" warning, this is because for this lab we used a training certificate authority which is not trusted by our computer.
-![Chrome warning](./images/chrome-error.png)
-Type "this is unsafe" on your keyboard (don't worry you won't see any letters appear), this will bypass the warning screen and you will be presented with our "Hello World" service.
-In the top bar click on "Not Secure" then click on "Certificate", here you can see the info about the certificate which we just issued.
-![Chrome certificate details](./images/chrome-cert.png)
+Once we see here that our certificate is issued by the Venafi TPP instance we can do the same describe on our secret resource that we asked cert-manager to put the certificate into:
```

Comment:
`describe on our secret ...` -> `describe on the Secret ...`

('the' and 'Secret')